org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "base": "io.atom.electron.BaseApp",
+    "separate-locales": "false",
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host",
+	"--socket=pulseaudio"
+    ],
+    "build-options" : {
+        "env": {
+            "NPM_CONFIG_LOGLEVEL": "info"
+        }
+    },
+    "modules": [
+        {
+            "name": "nuclear",
+            "buildsystem": "simple",
+            "sources": [
+               {
+                   "type": "archive",
+                   "url": "https://github.com/nukeop/nuclear/releases/download/v0.6.3/nuclear-fca030.tar.gz",
+                   "sha256": "5b90a1064736c158f62ea74c5814dffdd25f8b6a3877dfeed065377e744cdd2e"
+                },
+                {
+                    "type":"file",
+                    "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.metainfo.xml",
+                    "sha256":"17f2d8221f3cfec579457c05a4a56b85dcff2dd44bc86d5a2bca85e088af7c39",
+                    "dest-filename":"org.js.nuclear.Nuclear.metainfo.xml"
+                },
+                {
+                  "type":"file",
+                  "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.desktop",
+                  "sha256":"61256cb50cbfa54d93b9c344f4795f42f7dc73cd195be17915403d0ac281ce88",
+                  "dest-filename":"org.js.nuclear.Nuclear.desktop"
+                },
+                {
+                    "type":"file",
+                    "url":"https://github.com/nukeop/nuclear/raw/master/build/512.png",
```

Comment:
copy this one too in the repository.

you can remove it when the new upstream has them.

Reply:
okay